src/domain/swap/index.ts

```diff
@@ -0,0 +1,23 @@
+export * from "./swap-out-checker"
+
+export enum SwapProvider {
```

Comment:
IIRC we're avoiding enum. is that accurate @vindard? do you remember why?

Reply:
https://github.com/GaloyMoney/galoy/blob/64d67269dbc343035d6f36075e5e2b699f694b45/CONTRIBUTING.md#enums-via-object-constants

Reply:
cool. i fixed them according to the documentation